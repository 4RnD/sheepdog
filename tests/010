#!/bin/bash

seq=`basename $0`
echo "QA output created by $seq"

here=`pwd`
tmp=/tmp/$$
status=1        # failure is the default!

# get standard environment, filters and checks
. ./common.rc
. ./common.filter

_cleanup

$SHEEP $STORE/0 -z 0 -p 7000 -c $DRIVER
$SHEEP $STORE/1 -z 1 -p 7001 -c $DRIVER

sleep 2

$COLLIE cluster format -c 2
$COLLIE cluster recover disable

qemu-img create sheepdog:test 4G

# create 20 objects
for i in `seq 0 19`; do
    $COLLIE vdi write test $((i * 4 * 1024 * 1024)) 512 < /dev/zero
done

$SHEEP $STORE/2 -z 2 -p 7002 -c $DRIVER

# overwrite the objects
for i in `seq 0 19`; do
    $COLLIE vdi write test $((i * 4 * 1024 * 1024)) 512 < /dev/zero
done
