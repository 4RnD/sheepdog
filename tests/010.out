QA output created by 010
using backend farm store
Cluster recovery: disable
/tmp/sheepdog/0/obj/007c2b2500000000
/tmp/sheepdog/0/obj/007c2b2500000001
/tmp/sheepdog/0/obj/007c2b2500000002
/tmp/sheepdog/0/obj/007c2b2500000004
/tmp/sheepdog/0/obj/007c2b2500000005
/tmp/sheepdog/0/obj/007c2b2500000007
/tmp/sheepdog/0/obj/807c2b2500000000
/tmp/sheepdog/1/obj/007c2b2500000000
/tmp/sheepdog/1/obj/007c2b2500000001
/tmp/sheepdog/1/obj/007c2b2500000002
/tmp/sheepdog/1/obj/007c2b2500000003
/tmp/sheepdog/1/obj/807c2b2500000000
/tmp/sheepdog/2/obj/007c2b2500000000
/tmp/sheepdog/2/obj/007c2b2500000003
/tmp/sheepdog/2/obj/007c2b2500000006
/tmp/sheepdog/2/obj/007c2b2500000007
/tmp/sheepdog/2/obj/807c2b2500000000
/tmp/sheepdog/3/obj/007c2b2500000004
/tmp/sheepdog/3/obj/007c2b2500000005
/tmp/sheepdog/3/obj/007c2b2500000006
/tmp/sheepdog/3/obj/007c2b2500000007
/tmp/sheepdog/4/obj/007c2b2500000001
/tmp/sheepdog/4/obj/007c2b2500000002
/tmp/sheepdog/4/obj/007c2b2500000003
/tmp/sheepdog/4/obj/007c2b2500000004
/tmp/sheepdog/4/obj/007c2b2500000005
/tmp/sheepdog/4/obj/007c2b2500000006
Cluster status: running

Cluster created at DATE

Epoch Time           Version
DATE      3 [127.0.0.1:7000, 127.0.0.1:7001, 127.0.0.1:7002]
/tmp/sheepdog/0/obj/007c2b2500000000
/tmp/sheepdog/0/obj/007c2b2500000001
/tmp/sheepdog/0/obj/007c2b2500000002
/tmp/sheepdog/0/obj/007c2b2500000004
/tmp/sheepdog/0/obj/007c2b2500000005
/tmp/sheepdog/0/obj/007c2b2500000007
/tmp/sheepdog/0/obj/807c2b2500000000
/tmp/sheepdog/1/obj/007c2b2500000000
/tmp/sheepdog/1/obj/007c2b2500000001
/tmp/sheepdog/1/obj/007c2b2500000002
/tmp/sheepdog/1/obj/007c2b2500000003
/tmp/sheepdog/1/obj/807c2b2500000000
/tmp/sheepdog/2/obj/007c2b2500000000
/tmp/sheepdog/2/obj/007c2b2500000003
/tmp/sheepdog/2/obj/007c2b2500000006
/tmp/sheepdog/2/obj/007c2b2500000007
/tmp/sheepdog/2/obj/807c2b2500000000
/tmp/sheepdog/3/obj/007c2b2500000004
/tmp/sheepdog/3/obj/007c2b2500000005
/tmp/sheepdog/3/obj/007c2b2500000006
/tmp/sheepdog/3/obj/007c2b2500000007
/tmp/sheepdog/4/obj/007c2b2500000001
/tmp/sheepdog/4/obj/007c2b2500000002
/tmp/sheepdog/4/obj/007c2b2500000003
/tmp/sheepdog/4/obj/007c2b2500000004
/tmp/sheepdog/4/obj/007c2b2500000005
/tmp/sheepdog/4/obj/007c2b2500000006
a8775e30ddc5eda14d76e5361a514392  -
8124234db4755d4772c3440f2cbd1ecb  -
62630992718e77c73873ce1672d80786  -
19b9d4d75a5a71bfad364628880fa913  -
Cluster status: running

Cluster created at DATE

Epoch Time           Version
DATE      3 [127.0.0.1:7000, 127.0.0.1:7001, 127.0.0.1:7002]
/tmp/sheepdog/0/obj/007c2b2500000000
/tmp/sheepdog/0/obj/007c2b2500000001
/tmp/sheepdog/0/obj/007c2b2500000002
/tmp/sheepdog/0/obj/007c2b2500000004
/tmp/sheepdog/0/obj/007c2b2500000005
/tmp/sheepdog/0/obj/007c2b2500000006
/tmp/sheepdog/0/obj/007c2b2500000007
/tmp/sheepdog/0/obj/807c2b2500000000
/tmp/sheepdog/1/obj/007c2b2500000000
/tmp/sheepdog/1/obj/007c2b2500000001
/tmp/sheepdog/1/obj/007c2b2500000002
/tmp/sheepdog/1/obj/007c2b2500000003
/tmp/sheepdog/1/obj/007c2b2500000004
/tmp/sheepdog/1/obj/007c2b2500000005
/tmp/sheepdog/1/obj/007c2b2500000006
/tmp/sheepdog/1/obj/007c2b2500000007
/tmp/sheepdog/1/obj/807c2b2500000000
/tmp/sheepdog/2/obj/007c2b2500000000
/tmp/sheepdog/2/obj/007c2b2500000003
/tmp/sheepdog/2/obj/007c2b2500000004
/tmp/sheepdog/2/obj/007c2b2500000005
/tmp/sheepdog/2/obj/007c2b2500000006
/tmp/sheepdog/2/obj/007c2b2500000007
/tmp/sheepdog/2/obj/807c2b2500000000
/tmp/sheepdog/3/obj/007c2b2500000004
/tmp/sheepdog/3/obj/007c2b2500000005
/tmp/sheepdog/3/obj/007c2b2500000006
/tmp/sheepdog/3/obj/007c2b2500000007
/tmp/sheepdog/4/obj/007c2b2500000001
/tmp/sheepdog/4/obj/007c2b2500000002
/tmp/sheepdog/4/obj/007c2b2500000003
/tmp/sheepdog/4/obj/007c2b2500000004
/tmp/sheepdog/4/obj/007c2b2500000005
/tmp/sheepdog/4/obj/007c2b2500000006
Cluster recovery: enable
Cluster status: running

Cluster created at DATE

Epoch Time           Version
DATE      3 [127.0.0.1:7000, 127.0.0.1:7001, 127.0.0.1:7002]
/tmp/sheepdog/0/obj/007c2b2500000000
/tmp/sheepdog/0/obj/007c2b2500000001
/tmp/sheepdog/0/obj/007c2b2500000002
/tmp/sheepdog/0/obj/007c2b2500000003
/tmp/sheepdog/0/obj/007c2b2500000004
/tmp/sheepdog/0/obj/007c2b2500000005
/tmp/sheepdog/0/obj/007c2b2500000006
/tmp/sheepdog/0/obj/007c2b2500000007
/tmp/sheepdog/0/obj/807c2b2500000000
/tmp/sheepdog/1/obj/007c2b2500000000
/tmp/sheepdog/1/obj/007c2b2500000001
/tmp/sheepdog/1/obj/007c2b2500000002
/tmp/sheepdog/1/obj/007c2b2500000003
/tmp/sheepdog/1/obj/007c2b2500000004
/tmp/sheepdog/1/obj/007c2b2500000005
/tmp/sheepdog/1/obj/007c2b2500000006
/tmp/sheepdog/1/obj/007c2b2500000007
/tmp/sheepdog/1/obj/807c2b2500000000
/tmp/sheepdog/2/obj/007c2b2500000000
/tmp/sheepdog/2/obj/007c2b2500000001
/tmp/sheepdog/2/obj/007c2b2500000002
/tmp/sheepdog/2/obj/007c2b2500000003
/tmp/sheepdog/2/obj/007c2b2500000004
/tmp/sheepdog/2/obj/007c2b2500000005
/tmp/sheepdog/2/obj/007c2b2500000006
/tmp/sheepdog/2/obj/007c2b2500000007
/tmp/sheepdog/2/obj/807c2b2500000000
/tmp/sheepdog/3/obj/007c2b2500000004
/tmp/sheepdog/3/obj/007c2b2500000005
/tmp/sheepdog/3/obj/007c2b2500000006
/tmp/sheepdog/3/obj/007c2b2500000007
/tmp/sheepdog/4/obj/007c2b2500000001
/tmp/sheepdog/4/obj/007c2b2500000002
/tmp/sheepdog/4/obj/007c2b2500000003
/tmp/sheepdog/4/obj/007c2b2500000004
/tmp/sheepdog/4/obj/007c2b2500000005
/tmp/sheepdog/4/obj/007c2b2500000006
Cluster recovery: disable
Cluster status: running

Cluster created at DATE

Epoch Time           Version
DATE      8 [127.0.0.1:7000, 127.0.0.1:7001, 127.0.0.1:7002, 127.0.0.1:7003, 127.0.0.1:7004, 127.0.0.1:7005, 127.0.0.1:7006, 127.0.0.1:7007]
/tmp/sheepdog/0/obj/007c2b2500000000
/tmp/sheepdog/0/obj/007c2b2500000001
/tmp/sheepdog/0/obj/007c2b2500000002
/tmp/sheepdog/0/obj/007c2b2500000003
/tmp/sheepdog/0/obj/007c2b2500000004
/tmp/sheepdog/0/obj/007c2b2500000005
/tmp/sheepdog/0/obj/007c2b2500000006
/tmp/sheepdog/0/obj/007c2b2500000007
/tmp/sheepdog/0/obj/807c2b2500000000
/tmp/sheepdog/1/obj/007c2b2500000000
/tmp/sheepdog/1/obj/007c2b2500000001
/tmp/sheepdog/1/obj/007c2b2500000002
/tmp/sheepdog/1/obj/007c2b2500000003
/tmp/sheepdog/1/obj/007c2b2500000004
/tmp/sheepdog/1/obj/007c2b2500000005
/tmp/sheepdog/1/obj/007c2b2500000006
/tmp/sheepdog/1/obj/007c2b2500000007
/tmp/sheepdog/1/obj/807c2b2500000000
/tmp/sheepdog/2/obj/007c2b2500000000
/tmp/sheepdog/2/obj/007c2b2500000001
/tmp/sheepdog/2/obj/007c2b2500000002
/tmp/sheepdog/2/obj/007c2b2500000003
/tmp/sheepdog/2/obj/007c2b2500000004
/tmp/sheepdog/2/obj/007c2b2500000005
/tmp/sheepdog/2/obj/007c2b2500000006
/tmp/sheepdog/2/obj/007c2b2500000007
/tmp/sheepdog/2/obj/807c2b2500000000
e0b27e7466a3c21d0a4dedfed8bb9184  -
f35835c0a25be5ee75a536d1816c1db4  -
0faf5f38c28a38a6db1e6dfcdf259141  -
83bffbfb00dbcb7d6b4a2fa9274175b9  -
Cluster status: running

Cluster created at DATE

Epoch Time           Version
DATE      8 [127.0.0.1:7000, 127.0.0.1:7001, 127.0.0.1:7002, 127.0.0.1:7003, 127.0.0.1:7004, 127.0.0.1:7005, 127.0.0.1:7006, 127.0.0.1:7007]
/tmp/sheepdog/0/obj/007c2b2500000000
/tmp/sheepdog/0/obj/007c2b2500000001
/tmp/sheepdog/0/obj/007c2b2500000002
/tmp/sheepdog/0/obj/007c2b2500000003
/tmp/sheepdog/0/obj/007c2b2500000004
/tmp/sheepdog/0/obj/007c2b2500000005
/tmp/sheepdog/0/obj/007c2b2500000006
/tmp/sheepdog/0/obj/007c2b2500000007
/tmp/sheepdog/0/obj/807c2b2500000000
/tmp/sheepdog/1/obj/007c2b2500000000
/tmp/sheepdog/1/obj/007c2b2500000001
/tmp/sheepdog/1/obj/007c2b2500000002
/tmp/sheepdog/1/obj/007c2b2500000003
/tmp/sheepdog/1/obj/007c2b2500000004
/tmp/sheepdog/1/obj/007c2b2500000005
/tmp/sheepdog/1/obj/007c2b2500000006
/tmp/sheepdog/1/obj/007c2b2500000007
/tmp/sheepdog/1/obj/807c2b2500000000
/tmp/sheepdog/2/obj/007c2b2500000000
/tmp/sheepdog/2/obj/007c2b2500000001
/tmp/sheepdog/2/obj/007c2b2500000002
/tmp/sheepdog/2/obj/007c2b2500000003
/tmp/sheepdog/2/obj/007c2b2500000004
/tmp/sheepdog/2/obj/007c2b2500000005
/tmp/sheepdog/2/obj/007c2b2500000006
/tmp/sheepdog/2/obj/007c2b2500000007
/tmp/sheepdog/2/obj/807c2b2500000000
/tmp/sheepdog/4/obj/007c2b2500000001
/tmp/sheepdog/4/obj/007c2b2500000002
/tmp/sheepdog/6/obj/007c2b2500000003
/tmp/sheepdog/7/obj/007c2b2500000003
Cluster recovery: enable
Cluster status: running

Cluster created at DATE

Epoch Time           Version
DATE      8 [127.0.0.1:7000, 127.0.0.1:7001, 127.0.0.1:7002, 127.0.0.1:7003, 127.0.0.1:7004, 127.0.0.1:7005, 127.0.0.1:7006, 127.0.0.1:7007]
/tmp/sheepdog/0/obj/007c2b2500000000
/tmp/sheepdog/0/obj/007c2b2500000001
/tmp/sheepdog/0/obj/007c2b2500000002
/tmp/sheepdog/0/obj/007c2b2500000004
/tmp/sheepdog/0/obj/007c2b2500000005
/tmp/sheepdog/0/obj/807c2b2500000000
/tmp/sheepdog/1/obj/007c2b2500000000
/tmp/sheepdog/1/obj/007c2b2500000001
/tmp/sheepdog/1/obj/007c2b2500000002
/tmp/sheepdog/1/obj/807c2b2500000000
/tmp/sheepdog/2/obj/007c2b2500000000
/tmp/sheepdog/2/obj/007c2b2500000003
/tmp/sheepdog/2/obj/807c2b2500000000
/tmp/sheepdog/3/obj/007c2b2500000004
/tmp/sheepdog/3/obj/007c2b2500000005
/tmp/sheepdog/3/obj/007c2b2500000007
/tmp/sheepdog/4/obj/007c2b2500000001
/tmp/sheepdog/4/obj/007c2b2500000002
/tmp/sheepdog/4/obj/007c2b2500000004
/tmp/sheepdog/4/obj/007c2b2500000006
/tmp/sheepdog/5/obj/007c2b2500000005
/tmp/sheepdog/5/obj/007c2b2500000006
/tmp/sheepdog/5/obj/007c2b2500000007
/tmp/sheepdog/6/obj/007c2b2500000003
/tmp/sheepdog/7/obj/007c2b2500000003
/tmp/sheepdog/7/obj/007c2b2500000006
/tmp/sheepdog/7/obj/007c2b2500000007
